proto/friendship/friendship_service.proto

```
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package friendship;
+
+import "friendship/friendship_message.proto";
```

Buf (1.55.1)

import "friendship/friendship_message.proto": file does not exist

(COMPILE)